packages/pipeline/src/data_sources/ohlcv_external/crypto_compare.ts

```diff
@@ -0,0 +1,88 @@
+// tslint:disable:no-duplicate-imports
+import { AxiosResponse } from 'axios';
+import axios from 'axios';
+import promiseLimit = require('p-limit');
+
+import { merge, unfold } from 'ramda';
```

Comment:
There are some functions in Ramda that conflict with built-in functions or keywords. So sometimes we will have to import it with `import * as R from 'ramda'`. Could we also do that here to keep the name-spacing consistent? I also like being able to more easily see where the functions came from.